<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="4" hostname="LAPTOP-FK5U4VG9" name="com.qa.crm.testpages.RegisterPageTest" tests="7" failures="2" timestamp="2020-04-28T10:17:41 IST" time="234.326" errors="0">
  <testcase name="verifyTitleTest" time="26.424" classname="com.qa.crm.testpages.RegisterPageTest"/>
  <system-out/>
  <testcase name="verifyRegistration" time="37.087" classname="com.qa.crm.testpages.RegisterPageTest">
    <skipped/>
  </testcase> <!-- verifyRegistration -->
  <system-out/>
  <testcase name="verifyRegistration" time="42.327" classname="com.qa.crm.testpages.RegisterPageTest">
    <skipped/>
  </testcase> <!-- verifyRegistration -->
  <system-out/>
  <testcase name="verifyRegistration" time="29.701" classname="com.qa.crm.testpages.RegisterPageTest">
    <failure type="java.lang.AssertionError" message="expected [https://ui.freecrm.com/] but found [https://register.freecrm.com/register/]">
      <![CDATA[java.lang.AssertionError: expected [https://ui.freecrm.com/] but found [https://register.freecrm.com/register/]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.qa.crm.testpages.RegisterPageTest.verifyRegistration(RegisterPageTest.java:93)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- verifyRegistration -->
  <system-out/>
  <testcase name="verifyRegistration" time="30.715" classname="com.qa.crm.testpages.RegisterPageTest">
    <skipped/>
  </testcase> <!-- verifyRegistration -->
  <system-out/>
  <testcase name="verifyRegistration" time="36.627" classname="com.qa.crm.testpages.RegisterPageTest">
    <failure type="java.lang.AssertionError" message="expected [https://ui.freecrm.com/] but found [https://register.freecrm.com/register/]">
      <![CDATA[java.lang.AssertionError: expected [https://ui.freecrm.com/] but found [https://register.freecrm.com/register/]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.qa.crm.testpages.RegisterPageTest.verifyRegistration(RegisterPageTest.java:93)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- verifyRegistration -->
  <system-out/>
  <testcase name="verifyRegistration" time="31.445" classname="com.qa.crm.testpages.RegisterPageTest">
    <skipped/>
  </testcase> <!-- verifyRegistration -->
  <system-out/>
</testsuite> <!-- com.qa.crm.testpages.RegisterPageTest -->
